src/templates/select.hbs

```diff
@@ -30,8 +30,16 @@
     {{#if isLoading}}
       <span class="ember-select-loading">Loading...</span>
     {{/if}}
-    {{#if hasNoResults}}
-      <span class="ember-select-no-results">No results match "{{query}}"</span>
+    {{#if contentIsEmpty}}
+      <div class="ember-select-empty-content">
+        {{#if emptyContentView}}
```

Comment:
You can start your test the way it was before, then set the `emptyContentView` and check after that it's rendered